Dockerfile

```diff
@@ -1,20 +1,17 @@
 FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build
```

Comment:
I moved this file one directory level higher, based on this example of a project where one references the other:
https://github.com/ravimeda/dotnet-docker-samples/tree/ref-sample/ref-sample